Makefile

```diff
@@ -11,11 +11,11 @@ BINDIR= $(shell pwd)/bin
 VPATH = $(SRCDIR)
 
 CFLAGS=-c -g -Wall `root-config --cflags` -I${INCDIR}
-LDFLAGS=`root-config --glibs` -lHistPainter -lMinuit -L${ROOTSYS}/lib
+LDFLAGS=`root-config --glibs` -lHistPainter
```

Comment:
Was this on purpose? no other executables need these libraries?

Reply:
No not on purpose, it is only that locally if I wanted to run this, I had to take it off I think. But you do need this library for the ptf_qe_analysis

Reply:
OK, be careful when committing code to master, it should not break the system on `midptf`. Revert if necessary and commit directly to this branch. Or if you need, use some flags that determine what system you're compiling on to automatically set the necessary flags.